patterns/photo-blog-single-post-template.php

```diff
@@ -34,14 +34,14 @@
 						<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"constrained"}} -->
 						<div class="wp-block-group">
 							<!-- wp:paragraph {"fontSize":"small"} -->
-							<p class="has-small-font-size">Published on</p>
+							<p class="has-small-font-size"><?php esc_html__( 'Published on', 'twentytwentyfive' ); ?></p>
```

Comment:
[esc_html__()](https://developer.wordpress.org/reference/functions/esc_html__/) only retrieves the translation but doesn't print it. We should use either `echo esc_html__()` or [esc_html_e()](https://developer.wordpress.org/reference/functions/esc_html_e/)

Reply:
@juanfra There was some confusion on the functions at WCUS Contributor Day. I have updated them and re-requested review.